lang/ui.en.json

```diff
@@ -743,6 +747,18 @@
     "defaultMessage": "More edit in MakeCode options",
     "description": "Aria label for the additional actions menu to the right of the Edit in MakeCode button"
   },
+  "name-project": {
+    "defaultMessage": "Name your project",
+    "description": "Name your project header"
+  },
+  "name-text": {
+    "defaultMessage": "Name",
+    "description": "Header for name field"
+  },
+  "name-used-when": {
+    "defaultMessage": "The name is used when you save a file.",
+    "description": "Text under project name field"
+  },
```

Comment:
If we only ask for this when we save a file then this might not be worth it. In Python., the same dialog is used when you save a file with no name and when you edit a project's name. What do you think?

Reply:
True, it's redundant since we are only using it in one place and the dialog pops up part of the flow. Will remove it before I open PR for review

Reply:
No longer so sure about this... it feels weird to edit a name as a one-time thing so I think eventually we'll expose this elsewhere.

Reply:
Discussed this and agreed we'd plan to revisit (post-beta): collecting a name when starting a session & exposing the name and an edit feature.